app/facades/quote_facade.rb

```diff
@@ -0,0 +1,10 @@
+class QuoteFacade
+
+  def self.search_term(input)
+    json = QuoteService.search_by_input(input)
+
+    @quotes = json[:results].map do |quote_data|
+      QuotePoro.new(quote_data)
+    end
+  end
```

Comment:
In order to solve the issue of duplicate API calls, I would return either an array or a hash that includes all the data you want in the controller that comes from this one API call. That could look something like this:
```ruby
  def self.search_term(input)
    json = QuoteService.search_by_input(input)
    total = json[:totalCount]
    quotes = json[:results].map do |quote_data|
      QuotePoro.new(quote_data)
    end
    [total, quotes]
  end
```
and then in the controller, you would just update to:
```ruby
@total, @quotes = QuoteFacade.search_term(params[:query])
```